.github/workflows/land-blocking.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - name: Setup env
         run: |
           echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
+          echo "::set-env name=TEST_TAG::land_blocking_${LIBRA_GIT_REV}"
```

Comment:
nit - since this tag is actually user visible(it will be part of cti command printed to user) shorter the tag name the better.
So for example `land_xxx` would be more preferred then `land_blocking_xxx` - meaning is not lost but string looks shorter